acme/setup.py

```diff
@@ -27,6 +27,7 @@
     'pytest',
     'pytest-xdist',
     'tox',
+    'typing',
```

Comment:
Actually, we can technically remove this, since we're only running mypy under python3, where it's built in. And we don't need the import to actually be there otherwise, even in dev. Taking it out, lmk if you want it back in.